models/src_amazon_ads.yml

```diff
@@ -219,8 +219,8 @@ sources:
           - name: state
             description: "{{ doc('state') }}"
 
-      - name: targeting_keyword_report
-        description: Targeting reports contain performance metrics broken down by both targeting expressions and keywords.
+      - name: targeting_keyword_report 
```

Comment:
I see there is also a general `keyword_report` in the source. What is the difference between the `targeting_keyword_report` and the `keyword_report`? Why should we use the targeting one over the keyword?

Reply:
`keyword_report` is the v2 version of the keyword report, so I'm using `targeting_keyword_report` since it's v3. I found the best way to tell a v2 report is if the "last modified" date of the table in the details is not recent.

Reply:
Great callout! I didn't catch that when reviewing. This makes sense to me.